src/app/modules/layout/components/admin-layout/admin-layout.component.scss

```diff
@@ -36,6 +36,7 @@
   line-height: 64px;
   max-width: var(--sidenav-width); /*15rem;*/
   overflow: hidden;
+  padding-right: 6px;
```

Comment:
No need for padding as logo is already centered by way of the `justify-content: center` rule.

<img width="907" alt="image" src="https://user-images.githubusercontent.com/29865743/204879675-8c2baf99-0fe4-4245-9a2f-f136104214c0.png">

Reply:
It's centered programatically, not visually (easier to see on light themes).

Reply:
I see what you mean. If we want to correct this, we should also consider that the logo appears elsewhere in the UI as well. I'll approve and leave it up to you whether we want to include this here or do something more global on another ticket